lib/core/src/lib/toolbar/toolbar.module.ts

```diff
@@ -18,27 +18,13 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 import { TranslateModule } from '@ngx-translate/core';
-import { MaterialModule } from '../material.module';
 
 import { ToolbarDividerComponent } from './toolbar-divider.component';
 import { ToolbarTitleComponent } from './toolbar-title.component';
 import { ToolbarComponent } from './toolbar.component';
 
 @NgModule({
-    imports: [
-        CommonModule,
-        MaterialModule,
-        TranslateModule
-    ],
-    declarations: [
-        ToolbarComponent,
-        ToolbarTitleComponent,
-        ToolbarDividerComponent
-    ],
-    exports: [
-        ToolbarComponent,
-        ToolbarTitleComponent,
-        ToolbarDividerComponent
-    ]
+    imports: [CommonModule, ToolbarComponent, ToolbarTitleComponent, ToolbarDividerComponent, TranslateModule],
```

Comment:
I think you don't need anymore: CommonModule, TranslateModule here

Reply:
done